tests/neg/i1050c.scala

```diff
@@ -27,7 +27,8 @@ object Tiark4 {
     v.brand("boom!")
 }
 object V { // error: cannot be instantiated
```

Comment:
The error of the bound `>: Any <: Nothing` is the reason for the `cannot be instantiated` error, the full message is quite clear on the source of the issue.